Source/VMware.vSphereDSC/VMware.vSphereDSC.psd1

```diff
@@ -1,9 +1,9 @@
 <#
```

Comment:
Remove this file from the PR as Travis will take care of updating the content of it.